src/screens/Series/Series.tsx

```diff
@@ -141,6 +122,13 @@ const Series = ({ match, location }: RouteComponentProps<SeriesRouteParams>): JS
   const pageTitle = `${item.title} - ${siteName}`;
   const canonicalUrl = seriesPlaylist && item ? `${window.location.origin}${episodeURL(seriesPlaylist, item.mediaid)}` : window.location.href;
 
+  const primaryMetadata = formatVideoMetaString(t, item, true, seriesPlaylist.playlist.length);
+  const secondaryMetadata = (
+    <>
+      <strong>{formatSeriesMetaString(item)}</strong> - {item.title}
```

Comment:
What was the reason for not including the `strong` tag in the `VideoDetail` component?  Seems like unnecessary flexibility that might break the formatting if not used carefully.